service/tests/integration/build_crate.rs

```diff
@@ -1,64 +1,39 @@
 use std::path::{Path, PathBuf};
 
-use shuttle_service::loader::{build_crate, Runtime};
+use shuttle_service::builder::{build_crate, Runtime};
 
-#[tokio::test(flavor = "multi_thread")]
+#[tokio::test]
+#[should_panic]
```

Comment:
Let's add the expected error to this panic to make sure it is (and stays) correct

Reply:
It just gives a '1 job failed' error from the cargo compilation, I tried to check the error chain if there was more info but couldn't find any (I assume the compilation error is from not having a main function). So I added '1 job failed' to expected.